barretenberg/noir/bb_proof_verification/src/lib.nr

```diff
@@ -0,0 +1,43 @@
+pub global ULTRA_HONK_IDENTIFIER: u32 = 1;
+pub global ULTRA_HONK_VK_SIZE: u32 = 112;
+pub global ULTRA_HONK_PROOF_SIZE: u32 = 456;
+
+pub type UltraHonkProof = [Field; ULTRA_HONK_PROOF_SIZE];
+pub type UltraHonkVerificationKey = [Field; ULTRA_HONK_VK_SIZE];
+
+pub global ROLLUP_HONK_IDENTIFIER: u32 = 5;
+pub global ROLLUP_HONK_PROOF_SIZE: u32 = 466 + 69;
+pub global ROLLUP_HONK_VK_SIZE: u32 = 113;
```

Comment:
Is rolluphonk the proper name?

Reply:
One could argue whether its a good name but it does follow convention so for now thats good enough